bundles/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/internal/handler/FroniusSymoInverterHandler.java

```diff
@@ -12,8 +12,9 @@
  */
 package org.openhab.binding.fronius.internal.handler;
 
-import java.util.Objects;
+import java.util.Map;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Since we want to get rid of Apache Commons, can you replace this with native Java code? See #7722.